### PR TITLE
[W5.4][T13-A1]Pujitha Desiraju

### PR DESCRIPTION
**Pull request by Pujitha97**
Add update feature to update name,phone and email

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -10,9 +10,9 @@
  */
 public class Person implements ReadOnlyPerson {
 
-    private Name name;
-    private Phone phone;
-    private Email email;
+    public Name name;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Person.java:13]: try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.

**Review by okkhoy — COMMENTED** [review]
- Please note that you need to update the help as well next time.
- It would be good if you added tests for the Update function/feature that you implemented.

**Comment by okkhoy** [comment]
@Pujitha97 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/Person.java (hunk 1 above)
Comment (Pujitha97) [reply src/seedu/addressbook/data/person/Person.java:13]: Okay will try to incorporate that into my code next time.

**Comment by Pujitha97** [comment]
Comments noted.

**Pull request closed without merging**